lucien-ios/Protocols/CameraViewDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CameraViewDelegate.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/6/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+protocol CameraViewDelegate: class {
```

Comment:
This doesn't need to be in a separate file since it's small and strongly associated with `CameraViewController`. It can just go in the same file as it. Also, this naming is misleading. This is not a delegate for a `CameraView`, it's one for a `CameraViewController` and should be named accordingly.